src/adapter.ts

```diff
@@ -22,19 +22,28 @@ export function toGuidelimeStep(classicWowStep: Array<string>): GuidelimeStep {
       .map(s => s.charAt(0).toUpperCase() + s.substring(1))
       .join(' ');
 
+  const sanitizeCoords = (coords: string): string => {
```

Comment:
Add a regression test for the case that required this in `adapter.test.ts`. From what I understand it's the coords having a tilde at the front.